examples/certificates/cert.pem

```diff
@@ -1,11 +1,11 @@
 -----BEGIN CERTIFICATE-----
```

Comment:
we should remove this file completely and only ever have the user generate their own

Reply:
I'd prefer to keep it for now to avoid users the friction of having to generate their own